generator/internal/service_code_generator.h

```diff
@@ -109,6 +109,17 @@ class ServiceCodeGenerator : public GeneratorInterface {
    */
   bool HasStreamingReadMethod() const;
 
+  /**
+   * Determines if any of the method signatures has any Protocol Buffer
+   * Well-Known Types per
+   * https://developers.google.com/protocol-buffers/docs/reference/google.protobuf
+   * and provides the necessary include path.
+   * These additional includes are typically only necessary for building in
```

Comment:
I think these includes should always be necessary because we'd like to be include-what-you-use clean, if possible. So I think you can remove this last sentence.